lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -251,6 +258,20 @@ system_default_switchport_shutdown:
   config_get_token: ['/^system default switchport shutdown$/']
   # default_value: n/a. This is a user-configurable system default.
 
+vlan_mapping:
+  _exclude: [/N(3|5|6|9)K/]
```

Comment:
Do we need to account for `N3` without the `K` like we have in other areas?

https://github.com/cisco/cisco-network-node-utils/blob/f5816a4e559afd1e9365845df2ceff3f620cd8da/lib/cisco_node_utils/cmd_ref/fabricpath.yaml#L4